src/test/resources/test.template.properties

```diff
@@ -43,17 +43,18 @@ test.firefox.path=
 test.timeout=15
 
 ###############################################################################
-# Given below are the test accounts/passwords used for testing. 
+# Given below are the test accounts/passwords used for testing.
 # The values given below will work for the dev server(except one test case GodModTest).
 # To pass GodModTest, change ALL the accounts(except test.admin.account) by appending to the back of it.
 # e.g. alice.tmms -> alice.tmms.asd
-# If you plan to run the test suite against a staging server, 
-# you need to create four Google accounts (to use as test accounts) 
+# If you plan to run the test suite against a staging server,
+# you need to create four Google accounts (to use as test accounts)
 # and give their details below.
 
 # The Google account of a user that has 'admin access' to the application
 # For testing against staging server, you should use your own google account here.
-# Omit '@gmail.com'
+# Omit '@gmail.com' and use the commented out Id when not testing against staging server.
```

Comment:
Sorry, look here, this is the only useful part, the rest were all removed because my editor removes all extraneous spaces when I save the files.